Formula/ansilove.rb

```diff
@@ -0,0 +1,19 @@
+class Ansilove < Formula
+  desc "ANSi / ASCII art to PNG converter in C"
+  homepage "https://www.ansilove.org/"
+  url "https://github.com/ansilove/ansilove/archive/3.0.8.tar.gz"
+  sha256 "655472d31a9ec5a5ba29f5c15485d8647526ff8b0c9968d71f4267a9aa9bb544"
+
+  depends_on "cmake" => :build
+  depends_on "gd"
+
+  def install
+    system "cmake", "."
+    system "make"
+    bin.install "ansilove"
```

Comment:
Is there a `make install` target that could do that for us?

Reply:
Sorry I'm not much of a C developer.  There is a make install but I always get file permission errors when I do that locally?  I'm guessing I need to set the correct install destination but reading the Makefile I don't really understand how to do that :(

output when I use make install
```
CMake Error at cmake_install.cmake:36 (file):
  file INSTALL cannot copy file
  "/tmp/ansilove-20180409-93980-1hm6m3i/ansilove-3.0.8/ansilove" to
  "/usr/local/bin/ansilove".
```